src/DynamoCore/DynamoCore.csproj

```diff
@@ -95,6 +95,7 @@ limitations under the License.
     <Reference Include="System.Runtime.Serialization" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
+    <Reference Include="WindowsBase" />
```

Comment:
Just sanity check this is indeed need for?

Reply:
probably Application class
https://docs.microsoft.com/en-us/dotnet/api/system.windows?view=net-5.0

Reply:
it looks like this binary is actually cross platform despite it's name - but I would also like to know why we need it - we should avoid adding unnecessary dependencies, especially to DynamoCore.

Reply:
Removed this and encountered no issues.